entrypoint.sh

```diff
@@ -15,19 +15,24 @@ set -x
 echo "Getting anaconda token from github secrets..."
 
 ANACONDA_ORG="scientific-python-nightly-wheels"
-ANACONDA_TOKEN="$INPUT_ANACONDA_NIGHTLY_UPLOAD_TOKEN"
+ANACONDA_TOKEN="${INPUT_ANACONDA_NIGHTLY_UPLOAD_TOKEN}"
 
 # if the ANACONDA_TOKEN is empty, exit with status -1
 # this is to prevent accidental uploads
-if [ -z "$ANACONDA_TOKEN" ]; then
+if [ -z "${ANACONDA_TOKEN}" ]; then
   echo "ANACONDA_TOKEN is empty , exiting..."
   exit -1
 fi
 
+export ANACONDA_CLIENT_VERSION="1.12.0"
```

Comment:
Using `v1.12.0` fixes the problems in https://github.com/Anaconda-Platform/anaconda-client/issues/636.